robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
And lets chat about fetching values in UI when you have time btw. Hosts are probably only the second entity with `fetch_host_parameters` helper. We have multiple different implementations of the same basic operation for all the UI entities, imo it's time to standardize it and use single approach for all the entities (similar to `search`).